common/utils/src/main/scala/org/apache/spark/ErrorClassesJSONReader.scala

```diff
@@ -53,7 +53,8 @@ class ErrorClassesJsonReader(jsonFileURLs: Seq[URL]) {
         messageTemplate, "\\$\\{$1\\}"))
     } catch {
       case _: IllegalArgumentException => throw SparkException.internalError(
-        s"Undefined error message parameter for error class: '$errorClass'. " +
+        s"Undefined error message parameter for error class: '$errorClass', " +
+          s"MessageTemplate: $messageTemplate, " +
           s"Parameters: $messageParameters")
```

Comment:
can we also put the original exception in the causedBy field?

Reply:
ah good point. done